src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -835,9 +835,19 @@ private function addValidationSection(ArrayNodeDefinition $rootNode)
                                 ->end()
                             ->end()
                         ->end()
-                        ->booleanNode('disable_not_compromised_password')
-                            ->defaultFalse()
-                            ->info('Disable NotCompromisedPassword Validator: the value will always be valid.')
+                        ->arrayNode('not_compromised_password')
+                            ->addDefaultsIfNotSet()
+                            ->info('Non-compromised password validator configuration')
+                            ->children()
+                                ->booleanNode('enable')
+                                    ->defaultTrue()
+                                    ->info('Enable NotCompromisedPassword Validator.')
```

Comment:
the important part of the previous info was that disabling the validator would always considered the value as valid (i.e. as not compromised). You should keep that part in this info. Otherwise, it brings not value.

Reply:
Ok, I've added and adapted the previous info text.